src/network/networkd-address.c

```diff
@@ -326,7 +326,7 @@ static int address_add_foreign(Link *link, const Address *in, Address **ret) {
 
 static int address_add(Link *link, const Address *in, Address **ret) {
         Address *address;
-        int r;
+        int r, k;
```

Comment:
doesn't matter, but why not `bool is_new;` here?

i mean, bools neatly convert to ints, so if you do `return is_new` at the end, this still behaves exactly the same, but is just ~1.2 times prettier ;-)

Reply:
Done.